Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml

```diff
@@ -92,6 +92,15 @@
       - plasma
       - plasma_2
       - plasma_3
+  - type: SolutionContainer
+    caps: None
+    contents:
+      reagents:
+      - ReagentId: Plasma
+        Quantity: 10
+  - type: Tag
+    tags:
+    - Grindable
```

Comment:
Please don't contaminate your branches anymore.

Reply:
I don't even know how this happens ;-; sorry